websocket.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# OpenCraft -- tools to aid developing and hosting free software projects
+# Copyright (C) 2015 OpenCraft <xavier@opencraft.com>
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+"""
+Websocket server, based on Tornado & SwampDragon
+"""
+
+# Imports #####################################################################
+
+import os
+import sys
+
+from swampdragon.swampdragon_server import run_server
+
+
+# Main ########################################################################
+
+os.environ.setdefault("DJANGO_SETTINGS_MODULE", "opencraft.settings")
+
+host_port = sys.argv[1] if len(sys.argv) > 1 else 'localhost:2001'
```

Comment:
@antoviaque Can the host:port be pulled from django settings (and customizable in the `.env` file)? I'm trying to run it in a vagrant box without port forwarding.